src/askui/models/router.py

```diff
@@ -73,46 +73,46 @@ def __init__(self, log_level, report,
         self.huggingface_spaces = HFSpacesHandler()
         self.tars = UITarsAPIHandler(self.report)
     
-    def act(self, controller_client, instruction: str, model_name: str | None = None):
+    def act(self, controller_client, locator: str, model_name: str | None = None):
         if self.tars.authenticated and model_name == "tars":
-            return self.tars.act(controller_client, instruction)
+            return self.tars.act(controller_client, locator)
         if self.claude.authenticated and model_name == "claude":
             agent = ClaudeComputerAgent(controller_client, self.report)
-            agent.run(instruction)
+            agent.run(locator)
         raise AutomationError("Invalid model name for act")
```

Comment:
I think for `act` it is actually an instruction and not a locator.

Reply:
it's a goal